apps/finance/app/src/script.js

```diff
@@ -318,18 +337,19 @@ function loadTokenName(tokenContract) {
         .subscribe(
           name => {
             tokenName.set(tokenContract, name)
-            resolve(name)
+            resolve(name || fallback)
           },
           () => {
             // Name is optional
-            resolve('')
+            resolve(fallback)
```

Comment:
I just realized we don't return errors through aragonAPI's RPC... so a failing call here actually just returns `undefined` in the result handler. I think leaving these error handlers here is still good though :+1:.